### ISSUE TITLE
Accessibility ID of Element locator is not working

### ISSUE THREAD

**Issue by Priyaravi8** [rokwire/illinois-app#1281]
Element locator of accessibility ID of the **android.widget.Button of Building Access, Test location, MyMckinley, and Wellness Answer center** is not working when I use their accessibility id to locate the element for test automation scripting.
Other elements in the home screen like " Highlighted features and, Campus resources"work fine.

**Comment by TodorBachvarov** [comment]
Added in v3.2.15

**Comment by Priyaravi8** [comment]
This issue is still not fixed. The accessibility id of the element is not working. Element Attribute like "Id and Text" will also work.

**Comment by Priyaravi8** [comment]
@TodorBachvarov - I see a valid accessibility id and XPath. But when I locate the element with accessibility id/ xpath or check if the element is present in the screen using  -    **Page Should Contain Element    accessibility_id=Test Locations Find test locations**,  it is failing as it is not able to locate it.
Other accessibility ids/xpath in the home screen is working fine and I am able to use it in the script without any issues but not elements located under the "Building Access" section.

![Accessibility id](https://user-images.githubusercontent.com/26231179/157067645-42b2cc07-88e0-43cb-b9d2-c33c9591c169.png)

**Comment by TodorBachvarov** [comment]
@Priyaravi8, I've tried a few changes. Please try with version 3.3.9.

**Comment by knguyenjahiel** [comment]
Closing since this Building Access and (Covid) Testing Locations no longer relevant.
